docs/_tutorials/migration_v3.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migrating to V3
+order: 2
+slug: migration-v3
+lang: en
+layout: tutorial
+permalink: /tutorial/migration-v3
+---
+# Migrating to v3.x
+
+<div class="section-content">
+This guide will walk you through the process of updating your app from using `@slack/bolt@2.x` to `@slack/bolt@3.x`. There are a few changes you'll need to make but for most apps, these changes can be applied in 5 - 15 minutes.
+
+*Note: Make sure to checkout our [support schedule](#slackbolt1x-support-schedule) for `@slack/bolt@2.x` if you don't plan on upgrading right away*
+</div> 
+
+---
+
+### Org Wide App Installation Changes to InstallationStore & orgAuthorize
+
+In [Bolt for JavaScript 2.5.0](https://github.com/slackapi/bolt-js/releases/tag/%40slack%2Fbolt%402.5.0), we introduced support for [org wide app installations](https://api.slack.com/enterprise/apps). To add support to your applications, two new methods were introduced to the Installation Store used during OAuth, `fetchOrgInstallation` & `storeOrgInstallation`. With `@slack/bolt@3.x`, we have dropped support for these two new methods to better align with Bolt for Python and Bolt for Java. See the code samples below for the recommended changes to migrate.
```

Comment:
@stevengill How about mentioning not only alignment with other Bolt frameworks but also its simplicity or easiness of development? (please polish the sentence)
```suggestion
In [Bolt for JavaScript 2.5.0](https://github.com/slackapi/bolt-js/releases/tag/%40slack%2Fbolt%402.5.0), we introduced support for [org wide app installations](https://api.slack.com/enterprise/apps). To add support to your applications, two new methods were introduced to the Installation Store used during OAuth, `fetchOrgInstallation` & `storeOrgInstallation`. With `@slack/bolt@3.x`, we have dropped support for these two new methods for making the interface simpler and realizing better alignment with Bolt for Python and Bolt for Java. See the code samples below for the recommended changes to migrate.
```